src/tests/CMakeLists.txt

```diff
@@ -154,7 +154,7 @@ if(Python3_Interpreter_FOUND)
         )
         set_property(TEST non-regression PROPERTY LABELS non-regression sirius)
   else()   
```

Comment:
Instead of having an **if (numpy and pytest)** statement embracing a big block, we should have : 
```cmake
if (PYTHON_MODULE_pytest_FOUND AND PYTHON_MODULE_numpy_FOUND)
    message(FATAL_ERROR "Module pytest or numpy not installed : ..." )
```
Same remark for the : 
```cmake
if(Python3_Interpreter_FOUND)
```
as long as : 
```cmake
find_package(Python3 COMPONENTS Interpreter)
```
does not already do the job, of course : maybe **cmake** stops with an error if the **find_package** does not find anything.